### PR TITLE
Daml Finance: Adding changelog and release notes

### PR DESCRIPTION
**Pull request by johan-da**

### CODE REVIEW

File: docs/2.8.0/docs/daml-finance/packages/interfaces/changelogs/daml-finance-interface-instrument-equity.rst [hunk 1]
```diff
@@ -4,14 +4,27 @@
...
+
+- Renamed the `F` type synonym to `I`.
+
 Version 0.3.0
 *************
 
 - Update of SDK version and dependencies
 
-- The `Create` choice on the instrument factory returns the corresponding interface (rather than the base instrument interface)
+- The `Create` choice on the instrument factory returns the corresponding interface (rather than the
```
Comment (marciw-da) [inline docs/2.8.0/docs/daml-finance/packages/interfaces/changelogs/daml-finance-interface-instrument-equity.rst:24]: Here's another section where the list items don't have terminal punctuation; up to you whether to add it

File: docs/2.8.0/docs/daml-finance/packages/interfaces/changelogs/daml-finance-interface-instrument-generic.rst [hunk 2]
```diff
@@ -4,14 +4,28 @@
...
+
+- Renamed the `F` type synonym to `I`.
+
 Version 2.0.0
 *************
 
 - Update of SDK version and dependencies
 
-- The `Create` choice on the instrument factory returns the corresponding interface (rather than the base instrument interface)
+- The `Create` choice on the instrument factory returns the corresponding interface (rather than the
```
Comment (marciw-da) [inline docs/2.8.0/docs/daml-finance/packages/interfaces/changelogs/daml-finance-interface-instrument-generic.rst:24]: Another section without terminal punctuation -- I'm going to stop marking them now but maybe do an overall check if you do want to ensure consistency. Totally up to you.

**Review by marciw-da — COMMENTED** [review]
Made some suggestions. (Just the minimum since this kind of information isn't super readable anyway.) :) 

Do these changes need to be applied to the 3.0.0 docs directory as well? Will follow up over DM.

File: docs/2.8.0/docs/daml-finance/packages/interfaces/changelogs/daml-finance-interface-instrument-generic.rst (hunk 2 above)
Comment (johan-da) [reply docs/2.8.0/docs/daml-finance/packages/interfaces/changelogs/daml-finance-interface-instrument-generic.rst:24]: will do, thanks

**Comment by johan-da** [comment]
> Made some suggestions. (Just the minimum since this kind of information isn't super readable anyway.) :)
> 
> Do these changes need to be applied to the 3.0.0 docs directory as well? Will follow up over DM.

I applied the changes to 3.0.0 as well (in a separate commit).

**Review by marciw-da — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**